TESTING.md

```diff
@@ -117,3 +117,11 @@ DISPLAY=:99 mvn verify
 or specify the `DISPLAY` in the Eclipse JUnit launch configuration:
 
 ![junit_env_display.png](images/junit_env_display.png "screenshot of how to set custom DISPLAY")
+
+## Manual Testing
+### CMake Build Settings tab
+A set of manual tests that check it is possible to control the CMake build using the Launch Bar Launch Configuration > Build Settings tab.
+
+[CMake Build Settings](https://github.com/eclipse-cdt/cdt/files/14087582/Bug579242_manual_tests.md)
```

Comment:
I'm not sure how to edit the file in this link going forward. I also think I understand the question you were answering better now!

The link here goes to a file that is directly downloaded (e.g. open https://github.com/betamaxbandit/cdt/blob/Bug_579242/TESTING.md and click on the link)

> I'm not sure whether to add this file to the repo. If yes, then can someone suggest a suitable path please?
> How about?
> cdt/cmake/org.eclipse.cdt.cmake.ui.tests/manual_tests/

Yes - please add the file to the repo. Adding the file to that location is ok - with the link (relative please) being added to TESTING.md

Reply:
Thanks Jonah.

I've moved the Bug579242_manual_tests.md file into ./cmake/org.eclipse.cdt.cmake.ui.tests/manualTests and updated the TESTING.md file. The test link opens the page now, rather than downloads it.